apps/web/src/features/multichain/hooks/__tests__/useSafeCreationData.test.ts

```diff
@@ -1,21 +1,23 @@
-import { fakerChecksummedAddress, renderHook, waitFor } from '@/tests/test-utils'
-import { SAFE_CREATION_DATA_ERRORS, useSafeCreationData } from '../useSafeCreationData'
-import { faker } from '@faker-js/faker'
-import { PendingSafeStatus, type UndeployedSafe } from '@/store/slices'
 import { PayMethod } from '@/features/counterfactual/PayNowPayLater'
-import { chainBuilder } from '@/tests/builders/chains'
-import * as sdk from '@/services/tx/tx-sender/sdk'
-import * as cgwSdk from '@safe-global/safe-client-gateway-sdk'
 import * as web3 from '@/hooks/wallets/web3'
-import { encodeMultiSendData, type SafeProvider } from '@safe-global/protocol-kit'
+import * as sdk from '@/services/tx/tx-sender/sdk'
+import { PendingSafeStatus, type UndeployedSafe } from '@/store/slices'
+import { chainBuilder } from '@/tests/builders/chains'
+import { fakerChecksummedAddress, renderHook, waitFor } from '@/tests/test-utils'
 import { Safe__factory, Safe_proxy_factory__factory } from '@/types/contracts'
-import { type JsonRpcProvider } from 'ethers'
 import { Multi_send__factory } from '@/types/contracts/factories/@safe-global/safe-deployments/dist/assets/v1.3.0'
-import { type ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
+import { faker } from '@faker-js/faker'
+import { encodeMultiSendData, type SafeProvider } from '@safe-global/protocol-kit'
 import { EMPTY_DATA, ZERO_ADDRESS } from '@safe-global/protocol-kit/dist/src/utils/constants'
-import { getSafeSingletonDeployment, getSafeToL2SetupDeployment } from '@safe-global/safe-deployments'
+import * as cgwSdk from '@safe-global/safe-client-gateway-sdk'
+import { getMultiSendDeployment, getSafeSingletonDeployment, getSafeToL2SetupDeployment } from '@safe-global/safe-deployments'
+import { type ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
+import { type JsonRpcProvider } from 'ethers'
+import { SAFE_CREATION_DATA_ERRORS, useSafeCreationData } from '../useSafeCreationData'
 
 const setupToL2Address = getSafeToL2SetupDeployment({ version: '1.4.1' })?.defaultAddress!
+const multiSendAddress = getMultiSendDeployment({ version: '1.4.1' })?.defaultAddress!
+const safeSetupAddress = '0x18261ff68cC8D05EE046EDF5F4049B2077624a1d' // TODO: getSafeSetupDeployment({ version: '1.4.1' })?.defaultAddress
```

Comment:
As it is hosted in a different repo I am not sure this will ever become part of `safe-deployments`.
Maybe we could add a `constants.ts` file in the safenet feature folder, where we have the `SafenetSetupAddress` as exported constant.

Reply:
Sounds good!